news/2024/2024-11-14-osu-world-cup-2024-round-of-16-recap.md

```diff
@@ -0,0 +1,218 @@
+
+Tiebreakers have long been a sore spot for Russia. They were eliminated from each of the last three World Cup iterations via tiebreaker (and in fact hadn't won a tiebreaker **since 2011**), so they had to break their streak of tiebreaker losses to continue in the winners bracket. As a battle raged on in [AJITAMA Adventure OST](https://osu.ppy.sh/beatmapsets/2274953#osu/4847546)'s storyboard, another one also raged between the two teams. When the dust finally settled, it was Russia that emerged victorious by a margin of just 102,726 points. With their spot in the Quarterfinals secured, the Russian roster could breathe a sigh of relief.
+
+### ::{ flag=ES }:: Spain vs. ::{ flag=FI }:: Finland
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116148023)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2298526307)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Q3t9MojGc18?start=1132&end=1296" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Spain vs. Finland ended up being the star match in the loser's bracket last weekend — the only other match besides Russia vs. China that went down to the wire. The matchup was on an even playing field both seeding and skill-wise, which made for an extremely exciting match where only one pick brought a decisive victory with an 800k point difference for team Spain.
+
+Both teams were extremely confident in their strategies which ended up in a match where no breakpoints whatsoever occurred, the closest it got was during team Finland's [HD1](https://osu.ppy.sh/beatmapsets/2009101#osu/4179715) pick, ending with a razor-thin 71k difference between both teams. Some standout carry performances include ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899)'s 5x100 hidden FC on [FM2](https://osu.ppy.sh/beatmapsets/14309#osu/115384), and ::{ flag=ES }:: [creator](https://osu.ppy.sh/users/13962152)'s FC on [HD2](https://osu.ppy.sh/beatmapsets/1739433#osu/3555094) and near-FC on [HR2](https://osu.ppy.sh/beatmapsets/2274925#osu/4847498) that secured the tiebreaker for team Spain.
+
+Quite a bit was riding on this for both teams; Spain hadn't won an OWC match since 2016 (in fact, this was their best scoreline since 2018's group stage), and Finland hadn't missed a Quarterfinals weekend in OWC since 2019. Unfortunately for Spain, this Cinderella comeback story was not meant to be, as they fell to Finland by 90k points. However, Spain has definitely proved themselves to be a capable contender in future World Cups — keep an eye out for them!
```

Comment:
Quite a bit was riding on this match for both teams: Spain hadn't won an OWC match since 2016 (in fact, this was their best scoreline since 2018's group stage), while Finland hadn't missed a Quarterfinals weekend in OWC since 2019. Unfortunately for Spain, this Cinderella comeback story was not meant to be, as they fell to Finland on the tiebreaker by 90k points. However, Spain has definitely proved themselves to be a capable contender in future World Cups — keep an eye out for them!